### PR TITLE
Add Styles To Bookstore

### PR DESCRIPTION
**Pull request by Ogaga01**
## Hello Reviewer 👋 ,

in this project milestone, I implemented the following requirements:

- Add Styles to the bookstore

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Navbar = () => (
+
+  <nav>
+    <h1 className="header-one">Bookstore</h1>
+    <Link className="navlink" to="/">Books</Link>
+    <Link className="navlink" to="/Categories">Categories</Link>
+  </nav>
```
Comment (OLIPLICHE) [inline src/components/Navbar.js]: - [x] Please make sure all elements in the design were added. Kindly add the missing icon. 
ic

File: src/index.css [hunk 2]
```diff
@@ -1,41 +1,306 @@
...
+
+.display-book {
+  align-items: center;
+  display: flex;
+  width: 75rem;
+  height: 10.625rem;
+  margin-left: 6.25rem;
+  margin-top: 2.313rem;
+  background-color: #fff;
+}
```
Comment (OLIPLICHE) [inline src/index.css]: - [x] Please try and work on the margins and make your page looks like the original design.
- Your design 
b1
- Original design
-
b2

File: src/index.css [hunk 3]
```diff
@@ -1,41 +1,306 @@
...
+  margin-top: 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  opacity: 0.5;
+}
+
+.form-section {
+  margin-left: 6.25rem;
+  margin-top: 7rem;
+}
```
Comment (OLIPLICHE) [inline src/index.css]: - [x] Please there is a divider line on top of form section. Kindly implement that.
- The picture below is from the original design.
-
b3

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Ogaga01 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Linter is green (the last commit have a green check mark (✅)) 👏
README file is descriptive 👏
Commit messages are meaningful 👏 👏
Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @Ogaga01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**